src/test/kotlin/step2/CalculatorKtTest.kt

```diff
@@ -0,0 +1,40 @@
+package step2
+
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.assertThatThrownBy
+import org.junit.jupiter.api.Test
+
+class CalculatorKtTest {
+
+    @Test
+    fun `문자열을 계산한다`() {
+        val calculate = Calculator.calculate("2 + 3 * 4 - 10 / 2")
+
+        assertThat(calculate).isEqualTo(5.0)
+    }
+
+    @Test
+    fun `4개의 연산자를 통해 문자열을 계산한다`() {
+        val calculate = Calculator.calculate("2 + 3 * 4 / 2")
+
+        assertThat(calculate).isEqualTo(10.0)
+    }
+
+    @Test
+    fun `없는 연산자가 들어오면 예외를 발생한다`() {
+        assertThatThrownBy { Calculator.calculate("2 + 3 * 4 & 5 / 2") }
+            .isInstanceOf(IllegalArgumentException::class.java).hasMessage("Invalid operator: &")
+    }
+
+    @Test
+    fun `입력 값이 비어있으면 에외를 발생한다`() {
+        assertThatThrownBy { Calculator.calculate("") }
```

Comment:
```suggestion
        assertThatThrownBy { Calculator.calculate("       ") }
```

위와 같이 테스트를 바꾼다면 성공할까요? 

Emptry와 Blank의 차이에 대해 알아보고 변경해보면 어떨까요?

https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/is-empty.html

https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/is-blank.html

Reply:
좋은 것 같습니다.